packages/ethereum/ethereum-light-client/src/update.rs

```diff
@@ -55,12 +55,9 @@ pub fn update_consensus_state(
 
     if consensus_update.finalized_header.beacon.slot > current_consensus_state.slot {
         new_consensus_state.slot = consensus_update.finalized_header.beacon.slot;
-
         new_consensus_state.state_root = consensus_update.finalized_header.execution.state_root;
         new_consensus_state.storage_root = header.account_update.account_proof.storage_root;
-
-        new_consensus_state.timestamp = current_client_state
-            .compute_timestamp_at_slot(consensus_update.finalized_header.beacon.slot);
+        new_consensus_state.timestamp = consensus_update.finalized_header.execution.timestamp;
```

Comment:
The docs for this field needs to be updated, and the places we create consensus state as well (e2e + ethgenesis script)

Reply:
Updated docs, scripts already use the correct timestamp